src/components/dashboard/components/dictionary/DictionaryVersionsTable.jsx

```diff
@@ -17,7 +17,7 @@ const DictionaryVersionsTable = (version) => {
     <tr id="versiontable">
       <td>{id}</td>
       <td>{ (new Date(updated_on)).toLocaleDateString('en-US', DATE_OPTIONS)}</td>
-      <td><a href={version_url}>Browse in OCL</a> <Link className="subscription-link" onClick={() => { showSubModal(version_url); }} to="#">Subscription URL</Link></td>
+      <td><a href={version_url}>Browse in OCL</a> <Link className="downloadConcepts" onClick={download} to="#">Download</Link> <Link className="subscription-link" onClick={() => { showSubModal(version_url); }} to="#">Subscription URL</Link></td>
```

Comment:
This should be a button if it does not link anywhere as explained [here](https://github.com/openmrs/openmrs-ocl-client/pull/172#discussion_r224816322). I think a ticket should be created to address this.

Reply:
@dondrzzy Thank you for that observation and recommendation I have created the ticket here https://issues.openmrs.org/browse/OCLOMRS-260